docs/getting-started/launching-one-node/docker.md

```diff
@@ -9,51 +9,79 @@ title: Orion Docker
+### 2.1) Pull Orion Docker Image from Docker Hub
 
-To do that, you just have to run the following:
+Orion docker image is available in docker hub repository. To pull the image, execute the following command:
+```
+docker pull orionbcdb/orion-server
 ```
-docker run -it --rm -v $(pwd)/deployment/crypto/:/etc/orion-server/crypto -p 6001:6001 -p 7050:7050 orion-server
-``` 
-This command will store all the database data inside the running container, so that container termination will remove all data.
 
-If you plan more that just trying Orion, first you have to create __new__ and __unique__ cryptographic materials and optional, update Orion server configuration.   
+### 2.2) Build Your Own Orion Docker Image
 
-To create a minimal set of cryptographic materials, execute the `cryptoGen.sh` script as shown below.
-```
-./scripts/cryptoGen.sh deployment
+If you would like to build your own image rather than pulling it from the docker hub, execute the following steps:
+
+  1. Create the required directory using the command `mkdir -p github.com/hyperledger-labs`
+  2. Change the current working directory to the above created directory by issing the command `cd github.com/hyperledger-labs`
+  3. Clone the server repository with `git clone https://github.com/hyperledger-labs/orion-server`
+  4. Change the current working directory to the repository root directory by issuing `cd orion-server`
+  5. To build the docker image, issue the command `make docker` which will create a docker image named `orionbcdb/orion-server`
+
+## 3) Start Orion in a Docker Container
+
+To start a node, we need a certificate authority and crypto materials for the node and admin users. To simplify this task, we have provided sample
+crypto materials in `deployment/crypto/`.
+
+Let's start a node with the sample crypto materials:
+
+```docker
+docker run -it --rm -v $(pwd)/deployment/crypto/:/etc/orion-server/crypto -p 6001:6001 \
+    -p 7050:7050 orionbcdb/orion-server
 ```
-It will create a new set of crypto materials, including Certificate Authority certificate & key pair, _server_ certificate & key pair and certificate & key pairs for _admin_ and _user_ database users.
 
-#### Existing volumes
-As mentioned before, by default, the ledger & database data and the server configuration are stored inside the docker container, but it can be overwritten by mapping host folders to container volumes.
+Port `6001` is Orion REST API port and should be used to access Orion using its REST API or [Go SDK](https://github.com/hyperledger-labs/orion-sdk-go/)
+
+Port `7050` is used between nodes in a cluster for performing replication.
+
+:::info
+To change the port `6001` and `7050`, we need to change the configuration file and map the host directory holding this configuration to `/etc/orion-server/config`
+as explained next in [volumes](#volumes-for-pesistant-storage,-configuration,-and-crypto-materials).
```

Comment:
I didn't got how internal links are formed...

Reply:
Actually, I had used a wrong link. Now, I have updated it.